internal/pkg/service/stream/storage/level/local/diskwriter/network/connection/connection.go

```diff
@@ -238,3 +247,12 @@ func (m *Manager) isClosed() bool {
 		return false
 	}
 }
+
+func (c *connection) close(ctx context.Context) {
```

Comment:
Why `connection` contains manager of other connections? Looks like antipatern that this should be responsibility of manager instead of connection closure itself.

Reply:
@jachym-tousek-keboola , @Matovidlo Ok, so I understood from your comments that it needs one more iteration:
https://github.com/keboola/keboola-as-code/pull/1915/commits/a5f5acd87aeb3f6b299c938a1fdb0f788fabac01

`transport.Client/ClientConnection`:
- I added more method, so the `connection.Manager` doesn't need to store conn refs at all.

`connection.Manager` is responsible only for opening connections base on the `volumes` etcd mirror.
- So the code is much more simpler now, please chect it.

The tests are without modification.